agent/xds/routes.go

```diff
@@ -893,6 +896,12 @@ func (s *ResourceGenerator) makeRouteActionForSplitter(
 		return nil, fmt.Errorf("number of clusters in splitter must be > 0; got %d", len(clusters))
 	}
 
+	if 10000 < totalWeight {
```

Comment:
Nit: I'd suggest moving 10000 into a variable so its easier to see what it means/why its that value

Reply:
fixed